src/service/serviceStatus.js

```diff
@@ -0,0 +1,67 @@
+const fetch = require('node-fetch');
+const {Service} = require("./Service");
+
+const refreshDelay = 15;
+
+const environmentUrls = {
+    'AAT': buildNonProdUrl('aat'),
+    'PERFTEST': buildNonProdUrl('perftest'),
+    'ITHC': buildNonProdUrl('ithc'),
+    'DEMO': buildNonProdUrl('demo'),
+    'PROD': 'https://hmcts-access.service.gov.uk'
+}
+
+const services = {
+    'AAT': {
+        'idam-web-public': new Service('idam-web-public'),
+        'idam-api': new Service('idam-api')
+    },
+    'PERFTEST': {
+        'idam-web-public': new Service('idam-web-public'),
+        'idam-api': new Service('idam-api')
+    },
+    'ITHC': {
+        'idam-web-public': new Service('idam-web-public'),
+        'idam-api': new Service('idam-api')
+    },
+    'DEMO': {
+        'idam-web-public': new Service('idam-web-public'),
+        'idam-api': new Service('idam-api')
+    },
+    'PROD': {
+        'idam-web-public': new Service('idam-web-public'),
+        'idam-api': new Service('idam-api')
+    }
+};
+
+function buildNonProdUrl(env) {
+    return `https://idam-web-public.${env}.platform.hmcts.net`;
+}
+
+function getAllServiceStatus() {
+    return services;
+}
+
+(async function monitorStatus() {
+    for (const [envName, envUrl] of Object.entries(environmentUrls)) {
+        try {
+            const response = await fetch(envUrl + '/health');
+            const json = await response.json();
+
+            if(json.status === 'UP') {
+                services[envName]['idam-web-public'].setLastSeen(Date.now());
+            }
+
+            if(json.components.api.status === 'UP') {
+                services[envName]['idam-api'].setLastSeen(Date.now());
+            }
+
+        } catch (e) {
+        }
```

Comment:
This catch statement not doing anything here....may be just a simple console.log?

Reply:
I did have a console.log here, but then when the service is down, it will just keep posting a console message every 15s